src/middleware/withAuthorizedSpace.types.ts

```diff
@@ -1,10 +1,12 @@
 import * as Ucanto from '@ucanto/interface'
 import { Context as MiddlewareContext } from '@web3-storage/gateway-lib'
+import { GatewayIdentityContext as GatewayIdentityContext } from './withGatewayIdentity.types.js'
 
-export interface DelegationsStorageContext extends MiddlewareContext {
-  /** The identity of the gateway itself */
-  gatewayIdentity: Ucanto.Signer
+export interface DelegationsStorageContext
+  extends MiddlewareContext,
+    GatewayIdentityContext {
```

Comment:
**suggestion:** Rather than include `GatewayIdentityContext` in this, let's keep them separate and just add it to the intersection of context types coming out of the middleware. That way they're each their own thing.